lib/c-api/Cargo.toml

```diff
@@ -23,6 +23,8 @@ crate-type = ["cdylib", "rlib", "staticlib"]
 # We rename `wasmer` to `wasmer-api` to avoid the conflict with this
 # library name (see `[lib]`).
 wasmer-api = { version = "2.0.0", path = "../api", default-features = false, package = "wasmer" }
+wasmparser = { version = "0.78", default-features = false, optional = false }
+
```

Comment:
We can revert this, it shouldn't be needed

Reply:
Removed.